code/game/objects/machinery/overwatch.dm

```diff
@@ -323,41 +341,39 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 				return
 			selected.overwatch_officer = usr //Link everything together, squad, console, and officer
 			current_squad = selected
-			current_squad.message_squad("Attention - Your squad has been selected for Overwatch. Check your Game panel for objectives.")
-			current_squad.message_squad("Your Overwatch officer is: [operator.name].")
 			if(issilicon(usr))
 				to_chat(usr, span_boldnotice("Tactical data for squad '[current_squad]' loaded. All tactical functions initialized."))
 			visible_message(span_boldnotice("Tactical data for squad '[current_squad]' loaded. All tactical functions initialized."))
 			attack_hand(usr)
 		if("message")
 			if(current_squad && operator == usr)
-				var/input = stripped_input(usr, "Please write a message to announce to the squad:", "Squad Message")
+				var/input = tgui_input_text(usr, "Please write a message to announce to the squad:", "Squad Message")
 				if(input)
 					current_squad.message_squad(input, usr) //message, adds username
 					if(issilicon(usr))
 						to_chat(usr, span_boldnotice("Message sent to all Marines of squad '[current_squad]'."))
 					visible_message(span_boldnotice("Message sent to all Marines of squad '[current_squad]'."))
 		if("sl_message")
 			if(current_squad && operator == usr)
-				var/input = stripped_input(usr, "Please write a message to announce to the squad leader:", "SL Message")
+				var/input = tgui_input_text(usr, "Please write a message to announce to the squad leader:", "SL Message")
```

Comment:
Sidenote I noticed this has no IC filter
Todo check also what else stripped input does that tgui doesn't